docs/tutorials/kafka.ipynb

```diff
@@ -844,9 +844,9 @@
         "thread.start()\n",
         "\n",
         "for mini_ds in online_train_ds:\n",
```

Comment:
Note: you can use `ds.flat_map(lamnbda ds:ds)` to chain the nested datasets (`ds.interleave` works too).

Reply:
Will check and update in later revisions. Thanks for the info.